dragula.js

```diff
@@ -563,7 +563,13 @@ function never () { return false; }
 function always () { return true; }
 function getRectWidth (rect) { return rect.width || (rect.right - rect.left); }
 function getRectHeight (rect) { return rect.height || (rect.bottom - rect.top); }
-function getParent (el) { return el.parentNode === doc ? null : el.parentNode; }
+function getParent (el) { 
+  if (el) {
+    return el.parentNode === doc ? null : el.parentNode;
+  } else {
+    return null;
+  }
+}
```

Comment:
I think this would be easier to maintain
```suggestion
function getParent (el) { return el && el.parentNode !== doc ? el.parentNode : null; }
}
```